templates/connect-inject-deployment.yaml

```diff
@@ -25,6 +25,7 @@ spec:
         release: {{ .Release.Name }}
         component: connect-injector
     spec:
+      securityContext: {}
```

Comment:
Same question as previously-- if this is empty, does it need to be defined at all?